disnake/guild.py

```diff
@@ -3563,12 +3563,33 @@ async def kick(self, user: Snowflake, *, reason: Optional[str] = None) -> None:
         """
         await self._state.http.kick(user.id, self.id, reason=reason)
 
+    @overload
+    async def ban(
+        self,
+        user: Snowflake,
+        *,
+        clean_history_duration: Union[int, datetime.timedelta] = 86400,
+        reason: Optional[str] = None,
+    ) -> None:
+        ...
+
+    @overload
+    async def ban(
+        self,
+        user: Snowflake,
+        *,
+        delete_message_days: Literal[0, 1, 2, 3, 4, 5, 6, 7],
```

Comment:
this should have a default too, no?

Reply:
I expected that to result in issues since it'd have multiple matching overloads if you use just `ban(user)`, but that doesn't seem to be the case :p
https://github.com/DisnakeDev/disnake/pull/659/commits/0a818f9506d9ddaf1d838dc4bace59b6af31d42a